fails:Enumerable#take_while calls the block with initial args when yielded with multiple arguments
fails:Enumerable#take_while Enumerable with size when no block is given returned Enumerator size returns nil
fails:Enumerable#take_while Enumerable with no size when no block is given returned Enumerator size returns nil
